ansible_ai_connect/ai/api/urls.py

```diff
@@ -12,26 +12,12 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from django.urls import path
+from django.urls import include, path
 
-from .views import (
-    Chat,
-    Completions,
-    ContentMatches,
-    Explanation,
-    Feedback,
-    GenerationPlaybook,
-    GenerationRole,
-)
+from .versions.v0 import urls as v0_urls
+from .versions.v1 import urls as v1_urls
 
 urlpatterns = [
-    path("completions/", Completions.as_view(), name="completions"),
-    path("contentmatches/", ContentMatches.as_view(), name="contentmatches"),
-    path("explanations/", Explanation.as_view(), name="explanations"),
-    # Legacy
-    path("generations/", GenerationPlaybook.as_view(), name="generations"),
-    path("generations/playbook", GenerationPlaybook.as_view(), name="generations/playbook"),
-    path("generations/role", GenerationRole.as_view(), name="generations/role"),
-    path("feedback/", Feedback.as_view(), name="feedback"),
-    path("chat/", Chat.as_view(), name="chat"),
+    path("v0/", include((v0_urls, "v0"), namespace="v0")),
+    path("v1/", include((v1_urls, "v0"), namespace="v1")),
```

Comment:
Probably mean `v1` on L#22?

`include((v1_urls, "v1"),....)`

Reply:
thanks for catching